src/Presentation/DonationMembershipApplicationAdapter.php

```diff
@@ -51,4 +52,20 @@ private function getMembershipFormBankDataValues( PaymentMethod $paymentMethod )
 			'bankname' => $paymentMethod->getBankData()->getBankName(),
 		];
 	}
+
+	public function getInitialValidationState( Donation $donation ): array {
+		$validationState = [];
```

Comment:
Was it a conscious decision not start with null values for the validation keys? I agree that it should not make a difference?

Reply:
I left them out because I only want to override client-side values from the server side that I can be sure of. In the very unlikely case that the client changes its defaults, I don't have to change the server-side defaults.